climada/hazard/plot.py

```diff
@@ -158,6 +161,136 @@ def plot_intensity(
 
         raise ValueError("Provide one event id or one centroid id.")
 
+    @staticmethod
+    def plot_track_density(
```

Comment:
I am a bit confused now about this method. Why is it a Hazard method? This should be a `TCTracks` method right? It has nothing to do with Hazards in general. At most maybe it could be a `TropCyclone` method. 

In general, we should not add methods that have only a very specific narrow application to a general basis class. This will lead back to the clutter of methods that we now spend years to reduce. 

Where do you think this would fit best? `TCTracks` or `TropCyclones`? Since it is only about tracks and not about intensities, I would move this to `TCTracks`.